services/api/src/utils/tokens.js

```diff
@@ -1,11 +1,12 @@
 const jwt = require('jsonwebtoken');
-const crypto = require('crypto');
+
 const config = require('@bedrockio/config');
+const { nanoid } = require('nanoid');
```

Comment:
curious, is nanoid better here? we're using it here because we've already added it as a dependency for 2-factor yes?

Reply:
naturally because we already have the library 

but also because `crypto.randomBytes(16).toString('hex'); ` 
falls into 

```
Uniformity. random % alphabet is a popular mistake to make when coding an ID generator. The distribution will not be even; there will be a lower chance for some symbols to appear compared to others. So, it will reduce the number of tries when brute-forcing. Nano ID uses a [better algorithm](https://github.com/ai/nanoid/blob/main/index.js) and is tested for uniformity.
```

where with nano id, with 15 chars 

```
Speed: 1000 IDs per hour/second
~158 years needed, in order to have a 1% probability of at least one collision.
```
https://zelark.github.io/nano-id-cc/

I can weight the risks

Reply:
what does "random % alphabet" mean?

Reply:
I read that as an implementation that uses e.g randomBytes

Reply:
I'm curious about it but I'll just let it go and assume you're right for now :)

Reply:
all good, let me know if you think I am wrong, my point was mainly that i dont know how `crypto.randomBytes(16).toString('hex')` unique those ids really are gonna be